extensions/resteasy-reactive/quarkus-jaxrs-client/runtime/src/main/java/io/quarkus/resteasy/reactive/client/runtime/ToObjectArray.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.resteasy.reactive.client.runtime;
+
+import java.util.Collection;
+
+/**
+ * used by query param handling mechanism, in generated code
+ */
+@SuppressWarnings("unused")
+public class ToObjectArray {
```

Comment:
Super minor nitpick: This should probably be `final`

Reply:
forgot about it in the rework, will change in the next commit